src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -151,7 +167,12 @@ private void buildSscsDocumentFromScan(SscsCaseData sscsCaseData, State caseStat
                     List<SscsDocument> documents = new ArrayList<>();
 
                     if (!equalsIgnoreCase(scannedDocument.getValue().getType(), COVERSHEET)) {
+
                         SscsDocument sscsDocument = buildSscsDocument(sscsCaseData, scannedDocument, caseState);
+
+                        if (reinstatementFeatureFlag && REINSTATEMENT_REQUEST.getValue().equals(sscsDocument.getValue().getDocumentType())) {
```

Comment:
Is reinstatementFeatureFlag needed here?

Reply:
Not strictly but keeps state from being polluted